bin/zenrun.d/zenmib.sh

```diff
@@ -9,19 +9,9 @@
 ##############################################################################
 
 __DEFAULT__() {
-    local nocommit=0
-    if echo "$@" | egrep -- '--nocommit|--keeppythoncode'; then
```

Comment:
The help statement at line 20 still mentions `--nocommit` - I'm guessing that help needs to be updated also.